contents/blog/clickhouse-vs-snowflake.md

```diff
@@ -21,105 +21,111 @@ If I was to compare the databases to boats, ClickHouse is a jet ski – ultra-fa
-
-Snowflake was built for businesses with complex, ever-changing needs with big budgets. It’s by no means the fastest horse, but it is a very reasonable choice for a lot of companies that are just looking to get more out of their data.
+Snowflake was built for businesses with complex, ever-changing needs with big budgets. It’s by no means the fastest horse, but it’s the only one that can prance, race, dance, and even sing.
 
 ### What about Clickhouse?
 
-An obvious difference between Snowflake and ClickHouse is that ClickHouse is an open source solution that can be deployed on any environment: Metal, Cloud, K8s, you name it. ClickHouse doesn’t make money on any of those open source deployments, and this leads their team to be less sales-driven and more product lead (something we are huge fans of!). 
+An obvious difference between Snowflake and ClickHouse is that ClickHouse is an open source solution that can be deployed on any arbitrary server. ClickHouse doesn’t make money on any of those open source deployments, and this leads their team to be less sales-driven and more research-heavy. 
+
+The biggest difference is in Clickhouse’s strength. Let’s return to the analogy of ClickHouse being a jet ski. ClickHouse is fast. Incredibly, unbelievably fast. In particular, ClickHouse can return complex aggregations of terabyte-level data spanning millions of rows in milliseconds. In comparison, Snowflake takes dozens of seconds to query gigabyte-level data. 
+
+ClickHouse accomplishes this by optimizing the database for speed at returning aggregates. ClickHouse isn’t designed to be utilized like a stock PostgreSQL or mySQL database; it’s particularly good at write-heavy, mutation-low, read-and-reduce-heavy operations. It's perfect solution for analytics providers, like [PostHog](https://posthog.com/).
 
-The biggest difference is in Clickhouse’s strength. Let’s return to the analogy of ClickHouse being a jet ski. ClickHouse is fast. Incredibly, unbelievably fast. In particular, ClickHouse can return complex aggregations of terabyte-level data spanning millions of rows in milliseconds. In comparison, Snowflake takes dozens of seconds to query petabyte amounts data. 
+However, normal, on-prem ClickHouse lacks a lot of features that Snowflake likely considers table-stakes: 
 
-ClickHouse accomplishes this by optimizing the database for reading large swaths of data. ClickHouse isn’t designed to be utilized like a stock PostgreSQL or mySQL database; it’s particularly good at write-heavy, mutation-low, read-and-reduce-heavy operations. It's perfect solution for analytics providers, like [PostHog](https://posthog.com/).
+- A dedicated non-technical-friendly GUI for exploring and visualizing data
+- The ability to modify entry data without massive performance burns
+- The ability to scale architecture seamlessly with no migrations whatsoever
```

Comment:
I'm a bit unsure what the deal with these changes is. Are these bulllet points inaccurate? If so, fine, if not they should stay in.